Pages/data.js

```diff
@@ -0,0 +1,46 @@
+class Data{
+    constructor(){
```

Comment:
Why is there data.js in Pages here ?

Reply:
It was probably due to merge conflicts I had, I removed it on my local, and pushed new code